src/edgepi/pwm/edgepi_pwm.py

```diff
@@ -17,22 +17,20 @@
     PWM_MIN_DUTY_CYCLE,
 )
 
+class PwmDeviceError(Exception):
+    """Raised when PWM device doesn't exist"""
+
 class EdgePiPWM():
     """PWM module to provide PWM signal"""
     __pwm_pin_to_channel = {PWMPins.PWM1 : PWMCh.PWM_1,
                             PWMPins.PWM2 : PWMCh.PWM_2}
+    __pwm_1 = None
+    __pwm_2 = None
```

Comment:
> why are you using class variables instead of instance variables ?

this was a mistake, moved them to instance variable

I've re-defined them as dictionary instead, as you suggested